panel/src/components/Navigation/Link.vue

```diff
@@ -84,9 +88,16 @@ export default {
         return false;
       }
 
-      // don't route if it's an absolute link
-      if (typeof this.href === "string" && this.href.indexOf("://") > 0) {
-        return false;
+      if (typeof this.href === "string") {
+        // don't route if it's an absolute link
+        if (this.href.includes("://")) {
```

Comment:
What about protocol relative URLs? We could catch them here with an additional check “starts with `//`.

Reply:
✅